Modules/_ssl.c

```diff
@@ -2996,6 +2996,119 @@ static PyType_Spec PySSLSocket_spec = {
     .slots = PySSLSocket_slots,
 };
 
+#ifdef _MSC_VER
+static const unsigned char *
+_get_cert_bytes(const X509 *cert, int *length)
+{
+    unsigned char *cert_bytes;
+    int cert_bytes_length = i2d_X509(cert, NULL);
+    if (cert_bytes_length <= 0)
+    {
+        return NULL;
+    }
+
+    cert_bytes = PyMem_RawMalloc(cert_bytes_length);
```

Comment:
Is it possible to use pymalloc (`PyMem_Malloc`) here? Typically, we only need `RawMalloc` if we plan on using it inside `Py_BEGIN_ALLOW_THREADS` areas.

Reply:
Using PyMem_Malloc I got the following error in my sample program:
`Fatal Python error: _PyMem_DebugMalloc: Python memory allocator called without holding the GIL`

Reply:
I figured. It might be worth adding a comment mentioning that the GIL isn't held (so no Python APIs can get called).